### PR TITLE
created button component

### PR DESCRIPTION
**Pull request by sagargse**
created button component

### CODE REVIEW

File: cssfiles/buttons.css [hunk 1]
```diff
@@ -0,0 +1,71 @@
+.components-desc-button{
+    display:flex;
+    justify-content: center;
+    align-items: center;
+ 
+}
```
Comment (Rockyraven) [inline cssfiles/buttons.css:6]: remove this extra space

Comment (sagargse) [reply cssfiles/buttons.css:6]: ok i will

**Review by Shailseen — APPROVED** [review]
looks fine .You can procced to merge.

File: documentation.html [hunk 2]
```diff
@@ -231,6 +230,46 @@ <h2>Example of Badge with icon</h2>
                     </div>
                 </div>
                 <hr>
+              <div id="button">
+                    <div class="components-intro">
+                        <h1>Button</h1>
```
Comment (vishalg8454) [inline documentation.html:235]: H1 should remain exclusive for single use in a page. H2 can be used.

**Review by vishalg8454 — CHANGES_REQUESTED** [review]
Just a minor change and all good to merge.

File: documentation.html (hunk 2 above)
Comment (sagargse) [reply documentation.html:235]: ok will consider that

Comment (sagargse) [reply documentation.html:235]: ok will consider that

**Comment by sagargse** [comment]
> looks fine .You can procced to merge.

okay!

**Pull request merged**